.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 php:
-  - '5.5'
-  - '5.6'
```

Comment:
We can add few more v7 versions just to be safe: `7.1` and `7.3`

Reply:
Nice approach. Done.